src/Files.Backend/ViewModels/Shell/FuturisticShellPageViewModel.cs

```diff
@@ -1,22 +1,22 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using CommunityToolkit.Mvvm.ComponentModel;
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Messaging;
 using Files.Backend.ViewModels.Layouts;
 
 namespace Files.Backend.ViewModels.Shell
 {
     public sealed class FuturisticShellPageViewModel : ObservableObject
     {
-        public BaseLayoutViewModel ActiveLayoutViewModel
-        {
-            get;
-        }
+        public IMessenger Messenger { get; }
+
+        public BaseLayoutViewModel ActiveLayoutViewModel { get; private set; } // TODO(i): Send a message on active layout changed ActiveLayoutChangedMessage
 
         public BaseLayoutViewModel LeftLayoutViewModel { get; private set; }
 
         public BaseLayoutViewModel RightLayoutViewModel { get; private set; }
+
```

Comment:
@duke7553 I think it makes sense to account for this change in the future. Switching to a collection would potentially allow us to add support for multiple panes.

Reply:
I'd keep it as is for now. I want to make it easy to use collections in the future since we'll only be using `ActiveLayoutViewModel` for now and the rest will be broadcasted by `IMessnger`